server/Dockerfile

```diff
@@ -24,13 +30,13 @@ LABEL name="Infinispan Server" \
       io.openshift.tags="datagrid,java,jboss" \
       io.openshift.s2i.scripts-url="image:///usr/local/s2i/bin"
 
-# Download and extract the Infinispan Server
-USER root
+# Sha1sum
+ARG CHECK_HASH=true
```

Comment:
We can avoid using this parameter: if you don't have the SHA1, make the check to succeed regardless